scripts/express/app.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+const express = require('express');
+const path = require('path');
+const shell = require('shelljs');
+
+const app = express();
+const port = process.env.PORT || 8081;
+// npm script to build assets by webpack
+const buildCommand = 'npm run compile:prod';
```

Comment:
I think we still want to build out our review apps with a dev environment so we get the additional warnings. What do you think?